modxcms/revolution
# [3.x] Fix static file field visibility in legacy Element panel UIs

### What does it do?
Made a few small tweaks to ensure static file field is not visible when "Is Static" is not checked. Also, fixed the following small issues found while testing:
- When creating a new static Element and the default media source is "None," the static file field was showing the file browser instead of a plain text field
- When working with a non-static Template, the static file field would appear when changing the Media Source
- When working with a Template with no source assigned, the Media Source displayed a blank value instead of "None"

### Why is it needed?
To correct previously undetected bug introduced in #16082.

### How to test
Create and edit each element type and
- toggle from static to non-static to verify static file visibility is as-expected
- toggle between Media Sources to verify the file browser only becomes available for statics _not_ set to "None"

### Related issue(s)/PR(s)
Resolves #16096.
